erdpy/delegation/staking_provider.py

```diff
@@ -0,0 +1,141 @@
+import binascii
```

Comment:
Please add a `__init__.py` file in the `delegation` package, just like others have.

Reply:
done.